apps/condo/domains/news/schema/NewsItemScope.js

```diff
@@ -47,32 +48,6 @@ const NewsItemScope = new GQLListSchema('NewsItemScope', {
                 create: false,
                 update: false,
             },
-            hooks: {
-                resolveInput: async ({ operation, resolvedData, fieldPath }) => {
```

Comment:
Keystone processes hooks in the following order:
1) hooks from fields 
2) hooks from schema

Since a field `unitType` has transformations into `resolveInput` of field, and the value of the `type` field depends on it, this logic needs to be moved to the hooks of the schema